best-practices/README.md

```diff
@@ -104,11 +104,18 @@ Rails
 Testing
 -------
 
+(See also the rspec style guide that appears in this section)
+
+* First #describe what you are doing.
+* Then establish the #context.
+* #it only expects one thing (within reason).
+* Prefer explicitness - #it #its #specify #subject are nice shortcuts but
+consider whether they are sacrificing readability.
 * Avoid `any_instance` in rspec-mocks and mocha. Prefer [dependency injection].
-* Avoid `its`, `let`, `let!`, `specify`, `before`, and `subject` in RSpec.
```

Comment:
:+1:. I've bagged that post out elsewhere in detail. I think it's half strawman.